### PR TITLE
making changes so project compiles, modified student class, user clas…

### PR DESCRIPTION
**Pull request by jacktrig**
made changes to make sure everything will compile, changed ID type to UUID in endpoints

### CODE REVIEW

File: src/EndpointUser.java [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
   }
 
+
+
   /**
    * @return User[]
    */
   public User[] readList() {
-    return this.list;
+    return new Student[2];
```
Comment (josephaw1022) [inline src/EndpointUser.java]: This isn't the intention of the method. This method is supposed to return a list of Users (i.e a list of users).

Comment (jacktrig) [reply src/EndpointUser.java]: I just made an update to address. I think the return type here should be array list (over array) and then the "this.list" can be returned.

File: src/EndpointUser.java [hunk 2]
```diff
@@ -1,42 +1,48 @@
 import java.util.ArrayList;
+import java.util.*;
```
Comment (josephaw1022) [inline src/EndpointUser.java:2]: This isn't a good practice. While it is convenient. It can cause issues in unusual cases. So, only import what you need

**Comment by josephaw1022** [comment]
# This isn't what we're really looking for in the endpoints

- 1) You mixed the user and company classes up in the endpoints
- 2) The endpoint methods are done how Portia stated and showed to do it earlier this morning. 
- 3) The imports aren't done correctly (never import *)
- 4) Don't make a pr from the main branch of your fork
I can handle the intentions of this branch and PR

**Pull request closed without merging**